Frontend/speedcart-react/src/App.js

```diff
@@ -14,11 +15,29 @@ import NewShoppingList from '@pages/NewShoppingList';
 import ShoppingListDetail from '@pages/ShoppingListDetail';
 
 import './App.css';
+import TermsOfService from '@components/TermsOfService';
```

Comment:
Move import up with other component imports and rename component to "SitePolicies".